addons/pos_self_order/controllers/manifest_controller.py

```diff
@@ -0,0 +1,74 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import base64
+import json
+
+from odoo import http
+from odoo.http import request
+from odoo.tools import ustr, file_open
+
+
+class PosManifest(http.Controller):
+
+    @http.route('/pos-self/manifest/<config_id>/<access_token>', type='http', auth='public', methods=['GET'])
+    def posmanifest(self, config_id, access_token):
+        """ Returns a posmanifest describing the metadata associated with a web application.
+        Using this metadata, user agents can provide developers with means to create user
+        experiences that are more comparable to that of a native application.
+        """
+        company_name = request.env.company.name
+        manifest = {
+            'name': company_name,
+            'short_name': company_name,
```

Comment:
Instead of only using the company name for the icon when "downloading" the pwa, can we use the name of the shop as well? Something like: `Company Bar`

@dama-odoo

Reply:
You're right, User might be more interested by the config name then the company one. Thus why not starting with it, like `Bar (Company)` ?